server/src/server.ts

```diff
@@ -905,6 +908,13 @@ function onMessage(msg: p.Message) {
 
       if (initialConfiguration != null) {
         extensionConfiguration = initialConfiguration;
+        if (extensionConfiguration.path !== "" &&
+          extensionConfiguration.path[0] === "~"
```

Comment:
How about removing this? Not sure why one needs to special case the home directory.

Reply:
I tested it. If an absolute or relative path were used it is ok, but if the path starts with `~` then it's not working.

Reply:
I double-checked here. If remove this then paths that start with the home directory symbol are not working. Or do you suggest not handling this option at all? Or did I miss something?